.clang-tidy

```diff
@@ -42,5 +43,10 @@ performance-*,
 WarningsAsErrors:
 '*,
 -bugprone-narrowing-conversions,
+-bugprone-reserved-identifier,
 -bugprone-unused-raii,
--clang-diagnostic-deprecated-declarations'
+-clang-diagnostic-deprecated-declarations,
+-clang-analyzer-cplusplus.PlacementNew,
+-modernize-avoid-bind,
+' # TODO fix std::binds later
+# TODO Figure out if PlacementNew is a false positive
```

Comment:
> .../rapidjson/document.h:1143:21: error: Storage type is aligned to 1 bytes but allocated type is aligned to 8 bytes [clang-analyzer-cplusplus.PlacementNew,-warnings-as-errors]

Sigh. Not a false positive, no. Together with the RapidJson assert that prevents us from reliably running AddressSanitzer I'm starting to feel that we should drop RapidJson and replace it with something that is less of a dumpster fire...